pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -72,6 +72,17 @@ func (k *KogitoAppSpec) IsGitURIEmpty() bool {
 	return len(k.Build.GitSource.URI) == 0
 }
 
+// HasStandaloneKogitoResource checks if user has provided a single Kogito file resource to be deployed.
+func (k *KogitoAppSpec) HasStandaloneKogitoResource() bool {
+	if k == nil {
+		return false
+	}
+	if k.Build == nil || &k.Build.HasStandaloneKogitoResource == nil {
```

Comment:
Just a nitpick, IMHO the HasStandaloneKogitoResource can never be nil (because it is not a pointer, but value).

Reply:
correct, but & points to the memory address which can be null, no?

Reply:
& creates a pointer of a structure, true, but it points to the structure which it was created from :) so the structure exists.
I cannot image any case when `&k.Build.HasStandaloneKogitoResource` can be nil (if k.Build is not nil).

Reply:
probably :)

Reply:
will update.